apiserver/src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -66,7 +66,10 @@ public enum ConfigPropertyConstants {
     SCANNER_SNYK_API_VERSION("scanner", "snyk.api.version", "2022-11-14", PropertyType.STRING, "Snyk API version", ConfigPropertyAccessMode.READ_WRITE),
     SCANNER_SNYK_CVSS_SOURCE("scanner", "snyk.cvss.source", "NVD", PropertyType.STRING, "Type of source to be prioritized for cvss calculation", ConfigPropertyAccessMode.READ_WRITE),
     SCANNER_SNYK_BASE_URL("scanner", "snyk.base.url", "https://api.snyk.io", PropertyType.URL, "Base Url pointing to the hostname and path for Snyk analysis", ConfigPropertyAccessMode.READ_WRITE),
+    SCANNER_CSAF_ENABLED("scanner", "csaf.enabled", "false", PropertyType.BOOLEAN, "Flag to enable/disable CSAF Vulnerability Analysis", ConfigPropertyAccessMode.READ_WRITE),
+    SCANNER_CSAF_THRESHOLD("scanner", "csaf.threshold", "75", PropertyType.INTEGER, "Matching threshold for CSAF Vulnerability Analysis", ConfigPropertyAccessMode.READ_WRITE),
     VULNERABILITY_POLICY_FILE_LAST_MODIFIED_HASH("vulnerability-policy", "vulnerability.policy.file.last.modified.hash", null,  PropertyType.STRING, "Hash value or etag of the last fetched bundle if any", ConfigPropertyAccessMode.READ_ONLY),
+    VULNERABILITY_SOURCE_CSAF_ENABLED("vuln-source", "csaf.enabled", "false", PropertyType.BOOLEAN, "Flag to enable/disable CSAF Advisories", ConfigPropertyAccessMode.READ_WRITE),
```

Comment:
Should use `VulnDataSourceFactory#isDataSourceEnabled()` instead. A separate `enabled` property should not be necessary.